app/Resources/views/security/login.html.twig

```diff
@@ -0,0 +1,79 @@
+{% import ":other:macro.html.twig" as macro %}
+
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=EDGE">
+    <title>Atlassian Vendor Tools</title>
+    <link rel="stylesheet" href="{{ asset('assets/aui/css/aui.min.css') }}" media="all">
+    <link rel="stylesheet" href="{{ asset('assets/aui/css/aui-experimental.min.css') }}" media="all">
+    <link rel="stylesheet" href="{{ asset('assets/css/mpcrm.css') }}" media="all">
+    <link rel="shortcut icon" href="/bundles/app/favicon.ico" type="image/x-icon"/>
+    <link rel="icon" href="/bundles/app/favicon.ico" type="image/x-icon">
+</head>
+<body class="aui-page-fixed">
+<div id="page">
+    <header id="header" role="banner">
+        <nav class="aui-header" role="navigation" style="min-height: 40px">
+        </nav>
+    </header>
+    <section id="content" role="main">
+        <header class="aui-page-header">
+            <div class="aui-page-header-inner">
+                <div class="aui-page-header-main">
+                    <h1>{% block pageHeader %}Atlassian Vendor Tools{% endblock %}</h1>
+                </div>
+            </div>
+        </header>
+        <div class="aui-page-panel" style="min-height: 400px">
+            <div class="aui-page-panel-inner">
+                <section class="aui-page-panel-content">
+                    {% block body %}
+                        <div class="avt-login-wrap">
+                            {% if error %}
+                                <div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>
+                            {% endif %}
+
+                            <form action="{{ path("login") }}" method="post" class="aui">
+
+                                <div class="field-group">
+                                    <label for="username">Username<span class="aui-icon icon-required">(required)</span></label>
+                                    <input type="text" id="username" name="_username" value="{{ last_username }}"
+                                           required="required"
+                                           class="text medium-field"/>
+                                </div>
+
+                                <div class="field-group">
+                                    <label for="password">Password<span class="aui-icon icon-required">(required)</span></label>
+                                    <input type="password" id="password" name="_password" required="required"
```

Comment:
Please keep one tag formatting in the same line

Reply:
Applies to similar places as well

Reply:
Changed tag formatting in my each previous commits as you've required.